### PR TITLE
Added inventory, payment, simulation docs

### PR DESCRIPTION
**Pull request by KiLanger**

### CODE REVIEW

File: docs/docs/dev-manuals/services/inventory.mdx [hunk 1]
```diff
@@ -0,0 +1,135 @@
...
+## Repository Structure
+
+The repository is structured as follows:
+
+<div className="repository-structure" >
+
+- `/src/`: Source code of the service
+  - `events/`: Dapr event handling
+    - `dto/`: Event DTOs
+  - `health/`: GraphQL API
```
Comment (nk-coding) [inline docs/docs/dev-manuals/services/inventory.mdx]: I'm pretty sure there is no GraphQL API in `health`

File: docs/docs/dev-manuals/services/inventory.mdx [hunk 2]
```diff
@@ -0,0 +1,135 @@
...
+  - `shared/`: Shared Code between different modules
+    - `constants`: Shared constants
+    - `decorators`: Shared decorators
+    - `enums`: Shared enumerators
+    - `guards`: Shared guards
+    - `interfaces`: Shared interfaces
+    - `logger`: Logging configuration
+    - `scalars`: Shared graphQL scalars
+    - `utils`: Shared helpers
+- `.dapr`: Dapr Configuration
```
Comment (nk-coding) [inline docs/docs/dev-manuals/services/inventory.mdx]: in the other services we left these out, so either we add these config things there too or we remove them here

File: docs/docs/dev-manuals/services/inventory.mdx [hunk 3]
```diff
@@ -0,0 +1,135 @@
...
+- `.github`: Github Actions
+
+</div>
+
+## Business Processes
```
Comment (nk-coding) [inline docs/docs/dev-manuals/services/inventory.mdx]: if we want to have this, we should also add this section to the other services involved in BPs

Comment (KiLanger) [reply docs/docs/dev-manuals/services/inventory.mdx]: It's important. Other option: We put this into wherever the saga is descibed or in the Business Process themselves. I'll be sure to remove it for now.

**Review by nk-coding — APPROVED** [review]

**Pull request merged**